src/UID2.Client/UID2Encryption.cs

```diff
@@ -168,7 +167,45 @@ private static DecryptionResponse DecryptV3(byte[] encryptedId, IKeyContainer ke
             }
         }
 
-        internal static EncryptionDataResponse EncryptData(EncryptionDataRequest request, IKeyContainer keys, IdentityScope identityScope)
+        internal static EncryptionDataResponse Encrypt(string rawUid, KeyContainer keys, IdentityScope identityScope, DateTime now)
```

Comment:
should we make IdentityType a param and add it as a param for the public-facing UID2.Client.UID2Client.Encrypt method? currently it's not set and just default to email

Reply:
fixed by making IdentityType derived from rawUid